ci/terraform/variables.tf

```diff
@@ -19,9 +19,17 @@ variable "vcn_cidr" {
 }
 
 variable "director_subnet_cidr" {
-    default = "10.0.1.0/24"
+    default = "10.0.7.0/24"
 }
 
 variable "director_ad" {
     default  = "WZYX:PHX-AD-1"
 }
+
```

Comment:
Expose two new variables in the variables file to set the cidrs.

Reply:
The variables to set the cidrs are already exposed aren't they - director_subnet_cidr, bats_subnet1_cidr and bats_subnet2_cidr

Reply:
never mind...i got what you meant.